easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1152,18 +1153,110 @@ def _validate(self, attr, values):  # private method
         if self[attr] and self[attr] not in values:
             raise EasyBuildError("%s provided '%s' is not valid: %s", attr, self[attr], values)
 
+    def _handle_ext_module_metadata_by_probing_modules(self, dep_name, dependency=None):
+        """
+        helper function for handle_external_module_metadata
+        handles metadata for external module dependencies when there is not entry in the
+        metadata file
+
+        It should look for the pair of variables definitions in the available modules
+          1. CRAY_XXXX_PREFIX and CRAY_XXXX_VERSION
+          2. CRAY_XXXX_DIR and CRAY_XXXX_VERSION
+          2. CRAY_XXXX_ROOT and CRAY_XXXX_VERSION
+          5. XXXX_PREFIX and XXXX_VERSION
+          4. XXXX_DIR and XXXX_VERSION
+          5. XXXX_ROOT and XXXX_VERSION
+          3. XXXX_HOME and XXXX_VERSION
+
+        If neither of the pairs is found, then an empty dictionary is returned
+        """
+        if dependency is None:
+            dependency = dict()
+
+        short_ext_modname = dep_name.split('/')[0]
+        if not 'name' in dependency:
+            dependency['name'] = [short_ext_modname]
```

Comment:
the test is failing because this is added regardless of finding the rest of the metadata, if this is moved inside the `if dep_prefix and dep_version:` block then it doesn't create a problem

Reply:
fixed, thanks for pointing this out @migueldiascosta!